src/systems/thruster/Thruster.cc

```diff
@@ -321,6 +379,13 @@ void ThrusterPrivateData::OnCmdThrust(const msgs::Double &_msg)
 /////////////////////////////////////////////////
 double ThrusterPrivateData::ThrustToAngularVec(double _thrust)
 {
+  // Only update if the thrust coefficient was not set by configuration
+  // and angular velocity is not zero. Some velocity is needed to calculate
+  // the thrust coefficient otherwise it will never start moving.
+  if (!this->thrustCoefficientSet && this->propellerAngVel !=0)
```

Comment:
@marcoag could the thrust coefficient ever become zero while moving (in other words, could either $\alpha_1$ or $\alpha_2$ be negative)? If so, maybe we should reset thrust coefficient to one if propeller angular velocity ever becomes zero. WDYT?

Reply:
Also, I'm surprised the compiler didn't complain about a floating point comparison to zero. AFAIK `std::abs(this->propellerAngVel) < std::numeric_limits<double>::epsilon()` would be a slightly more robust test.

Reply:
It did threw a warning, I assumed anything that is not a strict zero would work. 

Changed to your suggestion: [51337cd](https://github.com/gazebosim/gz-sim/pull/1652/commits/51337cd036d004d73e14e96ecb362978e7089074).